Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -1720,6 +1720,41 @@ static void jsPDFDocFinalize(JSObjectRef object)
     [CATransaction commit];
 }
 
+IntPoint PDFPlugin::convertFromPluginToPDFView(const IntPoint& point) const
+{
+    return IntPoint(point.x(), size().height() - point.y());
+}
+
+IntPoint PDFPlugin::convertFromPDFViewToRootView(const IntPoint& point) const
+{
+    IntPoint pointInPluginCoordinates(point.x(), size().height() - point.y());
+    return valueOrDefault(m_rootViewToPluginTransform.inverse()).mapPoint(pointInPluginCoordinates);
+}
+
+IntRect PDFPlugin::convertFromPDFViewToRootView(const IntRect& rect) const
+{
+    IntRect rectInPluginCoordinates(rect.x(), rect.y(), rect.width(), rect.height());
+    return valueOrDefault(m_rootViewToPluginTransform.inverse()).mapRect(rectInPluginCoordinates);
```

Comment:
```suggestion
    return valueOrDefault(m_rootViewToPluginTransform.inverse()).mapRect(rect);
```

Nit - maybe we can just make this a bit cleaner while we're shuffling the code around?

Reply:
Not going to clean up the to-be-deleted code :)